client/src/containers/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import SignInForm from './SignInForm/SignInForm'
+import Grid from "@material-ui/core/Grid";
+
+const SignIn = () => {
+  return (
+    <Grid
+    container
+    direction="column"
+    alignItems="center"
+    justify="center"
+    style={{ minHeight: "100vh" }}
```

Comment:
Please, don't use inline styling. Better extend this component by styled-components